internal/pkg/manifest/workload.go

```diff
@@ -473,6 +475,41 @@ func (c *NetworkConfig) requiredEnvFeatures() []string {
 	return nil
 }
 
+// ServiceConnect represents ECS Service Connect configuration.
+type ServiceConnect struct {
```

Comment:
I thought about it but tbh i'd rather keep as is 🤯

Reply:
Because adding `ArgsOrBool` really adds no value. It might make more sense to differentiate if there's already a `ServiceConnect` struct in this package.

Reply:
I liked that for all the other `XorY` namings because they makes it clear that the struct is meant to have values in one or the other. It's just a nit though no biggeie/

Reply:
I personally don't like this format because I think this has the same problem if we name a list var to be `var subnetList []string`.

Reply:
but eventually across the team we need a naming consensus for this...